.gemini/settings.json

```
@@ -0,0 +1,5 @@
+{
+  "context": {
+    "fileName": ["AGENTS.md", "redhat-compliance-and-responsible-ai.md"]
+  }
+}
```

```suggestion
}

```

Could we add a newline at the end of the file?

---

Done!
Also, I added a new rule to force AI agents to add newlines to each source file they work with.